### ISSUE TITLE
Custom panel height

### ISSUE THREAD

**Issue by Wilzi** [akreitals/angular-panel-snap#5]
It would be great to use custom panel heights.
As I can see the scrollInterval value is set by panel group container's height and this value is multiplied by current panel index in snapToPanel function.
I think hardcoded inline CSS values in akPanel directive should set in outline CSS file.

Here is a JSbin to test the issue:
http://jsbin.com/niwikuvapi/1/edit?html,output

Thank you for this great plugin! :)
If I have some time to solve this problem I'll send a pull request

**Comment by akreitals** [comment]
Thanks for providing a JSbin right off the bat, always helpful, and PR's are always welcome :)

Allowing panels to be bigger would be a strange UX as it would snap to the next panel before you could see the bottom of the last. But I see the use case for smaller panels.

Just to scope this out a little further, a few questions. Should it be a requirement that the panel height be a submultiple of the container height? i.e. 50%, 33%, 25%, 12.5%, 10% etc? If yes would an attribute, say {{panelsPerScreen=2}} suffice? Else should any arbitrary height be allowed? Could the heights differ for each panel?

**Comment by ericodes** [comment]
I completely agree! Since sometimes a content of a 'page' is greater than the window height. Right now, with my work around, you scroll within the inner section first, and when you get to the bottom of the inner section, you continue scrolling through these angular-panels.

Would be great to be able to have min-height: 100% on each panel! :)
